.github/ISSUE_TEMPLATE/helm_bug_report.yaml

```diff
@@ -0,0 +1,56 @@
+---
+name: Ngrok Ingress Controller Helm Chart Bug Report
+labels: ["bug", "area/helm-chart", "needs-triage"]
+body:
+- type: markdown
+  attributes:
+    value: |
+      Thanks for taking the time to fill out this bug report!
+
+      Note, you do not need to create an issue if you have a change ready to submit.
+      You can open a [pull request](https://github.com/ngrok/ngrok-ingress-controller/pulls) immediately instead.
+- type: input
+  attributes:
+    label: Kubernetes Version
+    description: Which Kubernetes Version do you use?
+  validations:
+    required: true
+- type: input
+  attributes:
+    label: Helm Chart Version
+    description: Which version of the Helm chart do you use?
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Helm Chart configuration
+    description: Additional description of your Helm Chart configuration.
+    placeholder: >
+      Enter any relevant details of your Helm Chart configuration. Maybe you can
+      paste your `values.yaml` or important parts of it here? Make sure to surround the code
+      you paste with ``` ```.
```

Comment:
some of these (if not all) seem like they would also be useful for the controller bug report. I wonder how accurate these will be reported fwiw as many problems would probably be some combination of the 2?

Reply:
Yeah. I'm hoping some things like annotations on ingress objects will be clearly in the controller realm and things like the helm install not working going into the helm bug report. For things that are a combination of the two, I think we will definitely need to triage them appropriately.